build/Packaging.cake

```diff
@@ -2,17 +2,25 @@
 
 var version = Argument("version", "0.4.0");
 
+Task("Build-ObjC")
+    .Does(() =>
+    {
+        StartProcess("make", new ProcessSettings {
```

Comment:
I think you have to check the exit code and decide to fail or not with `StartProcess`